### PR TITLE
week-0-a

### PR DESCRIPTION
**Pull request by karenl7**
more edits

### CODE REVIEW

File: bite-size-notes/week-0-a-test.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Testing workflow
 
 Here are my notes
+
 more edits!
+
+and more edits!
```
Comment (karenl7) [inline bite-size-notes/week-0-a-test.md]: I am leaving a comment here.

File: bite-size-notes/week-0-a-test.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Testing workflow
 
 Here are my notes
+
```
Comment (karenl7) [inline bite-size-notes/week-0-a-test.md]: And another comment here.

**Review by karenl7 — COMMENTED** [review]
author of week-0-b has left comments

File: bite-size-notes/week-0-a-test.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
 more edits!
+
```
Comment (karenl7) [inline bite-size-notes/week-0-a-test.md]: another comment! thie equation is not correct.

File: bite-size-notes/week-0-b.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+## new notes
+dslkfjsldkjfls
```
Comment (ctrl-admin) [inline bite-size-notes/week-0-b.md:2]: Adding more comments here

File: bite-size-notes/week-0-a-test.md (hunk 1 above)
Comment (karenl7) [reply bite-size-notes/week-0-a-test.md]: Thanks for the feedback. I will address this in the updated version.

File: bite-size-notes/week-0-b.md (hunk 4 above)
Comment (karenl7) [reply bite-size-notes/week-0-b.md:2]: Incorporated this comment in the updated version.

**Pull request merged**